client/state/themes/selectors/get-recommended-themes.js

```diff
@@ -9,9 +15,12 @@ const emptyList = [];
  * Gets the list of recommended themes.
  *
  * @param {object} state Global state tree
+ * @param {boolean} blockThemes Whether or not the recommended themes are block themes
  *
  * @returns {Array} the list of recommended themes
  */
-export function getRecommendedThemes( state ) {
-	return state.themes.recommendedThemes.themes || emptyList;
+export function getRecommendedThemes( state, blockThemes = false ) {
```

Comment:
`NB:` How would you feel about renaming `blockThemes`? I keep thinking it's some sort of data structure, when, in reality, it's a boolean check. Maybe something like `shouldUseBlockThemes` instead?

Reply:
I will definitely do this if I don't just eliminate the booleans altogether based on @copons' feedback above. Much better clarity.